destination/config.go

```diff
@@ -0,0 +1,28 @@
+package destination
+
+//go:generate paramgen -output=paramgen.go Config
+
+type Config struct {
+	// amazon access key id
+	AWSAccessKeyID string `json:"aws.accessKeyId" validate:"required"`
+	// amazon secret access key
+	AWSSecretAccessKey string `json:"aws.secretAccessKey" validate:"required"`
+	// amazon sqs region
+	AWSRegion string `json:"aws.region" validate:"required"`
+	// amazon sqs queue name
+	AWSQueue string `json:"aws.queue" validate:"required"`
+	// amazon sqs message delay time
+	AWSSQSMessageDelay int32 `json:"aws.delayTime"`
```

Comment:
<img width="1029" alt="image" src="https://github.com/conduitio-labs/conduit-connector-sqs/assets/96070127/b68e6b92-1f43-4631-b708-abd435ffbb15">

Essentially a certain time frame in seconds we want to wait until the message shows up in SQS. I've seen this show up in examples of a record being sent to a queue, so I thought it would be good idea include it (we can omit it if needed)

Reply:
Ah, I should have googled.